rust/main/agents/relayer/src/msg/metadata/routing.rs

```diff
@@ -30,10 +31,57 @@ impl MetadataBuilder for RoutingIsmMetadataBuilder {
             .build_routing_ism(ism_address)
             .await
             .map_err(|err| MetadataBuildError::FailedToBuild(err.to_string()))?;
-        let module = ism
-            .route(message)
+
+        let message_domain = KnownHyperlaneDomain::try_from(message.origin)
```

Comment:
imo we should avoid `KnownHyperlaneDomain` here, let's just use `self.base_builder().origin_domain()` (or `message.origin` if that's not satisfactory for some reason)

https://hyperlaneworkspace.slack.com/archives/C08GR6PBPGT/p1744653023063519?thread_ts=1744652424.257009&cid=C08GR6PBPGT